<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditWidget</class>
 <widget class="QWidget" name="CEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Time">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>131</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="TimeType"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Upper:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="TimeUpper">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lower:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="TimeLower"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Notevalue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>391</width>
     <height>58</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>58</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="10,1">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="spacing">
     <number>-1</number>
    </property>
    <item row="0" column="0">
     <widget class="QMacButtons" name="NoteButtons" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QMacButtons" name="DotTripletButtons" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="Pianoframe">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>252</width>
     <height>152</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>252</width>
     <height>152</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>252</width>
     <height>152</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CAccidentalsPiano" name="Accidentals" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="EditgroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>48</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::WrapLongRows</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="EditLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Edit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="SpingroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>51</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::WrapLongRows</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="SpinLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="Spin">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>22</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="CombogroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>46</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout_6">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::WrapLongRows</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="ComboLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="Combo">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Tempo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QSpinBox" name="TempoSpin">
      <property name="minimum">
       <number>30</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QMacButtons" name="TempoButtons" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QMacButtons" name="TempoDotted" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Controller">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="ControllerLabel">
      <property name="text">
       <string>Controller:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ValueLabel">
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="ValueSpin"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="ControllerCombo"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="TextFrame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFontWidget" name="FontBox">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMacButtons</class>
   <extends>QWidget</extends>
   <header>qmacbuttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFontWidget</class>
   <extends>QGroupBox</extends>
   <header>qfontwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CAccidentalsPiano</class>
   <extends>QWidget</extends>
   <header>caccidentalspiano.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
